### PR TITLE
Fix for the problem that only the cab came out of the garage and not …

### PR DESCRIPTION
**Pull request by WirzRexTon**
…the right list + Local fix

### CODE REVIEW

**Review by jbdevic — APPROVED** [review]
good

File: client/main.lua [hunk 1]
```diff
@@ -165,8 +165,8 @@ function OpenVehicleSpawnerMenu()
                 return
             end
             ESX.TriggerServerCallback("esx_taxijob:SpawnVehicle", function()
-                ESX.ShowNotification(TranslateCap('vehicle_spawned'), "success")
-            end, "taxi", {plate = "TAXI JOB"})
+                ESX.ShowNotification(TranslateCap('vehicle_spawned', element.title), "success")
+            end, element.model, {plate = "TAXI JOB"})
```
Comment (Gellipapa) [inline client/main.lua:169]: If element.model is nil? Please handle: element.model or "taxi" 
Thank you.

**Review by Gellipapa — CHANGES_REQUESTED** [review]
@WirzRexTon Please read my message and add this changes.

**Comment by WirzRexTon** [comment]
@Gellipapa Good

**Comment by Gellipapa** [comment]
@WirzRexTon Thank you, Do you tested?

**Comment by WirzRexTon** [comment]
@Gellipapa Yes I tested it, a person on the discord tried to modify his code by mine and it works perfectly.

**Comment by WirzRexTon** [comment]
and I tested it myself in game

**Comment by Gellipapa** [comment]
#26 Thank you test, i handle some use case. Merge other pull request.

**Comment by WirzRexTon** [comment]
@Gellipapa Can we talk about something privately on discord please? Nickname: .wirzrexton

**Comment by Gellipapa** [comment]
@WirzRexTon Hi! Yes write me.

**Pull request closed without merging**